public_suffix_list.dat

```diff
@@ -15144,6 +15144,7 @@ shopitsite.com
 // shopware AG : https://shopware.com
 // Submitted by Jens Küper <cloud@shopware.com>
 shopware.store
+shopware.shop
```

Comment:
The sorting seems to be incorrect. `shopware.shop` should be positioned above `shopware.store` ([Guideline](https://github.com/publicsuffix/list/wiki/Guidelines#sort-your-submission-correctly-important))